sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,9 +1,26 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-10-08)
 
+### New Features
+
+- Added NOTICE.txt for 3rd party dependencies
+
+### Breaking Changes
+
+- Throw Azure::Http::TransportException if creating new connection fails
+- Response objects store Nullable<T>
+
+### Bug Fixes
+
+- Switched to a more stable wait on sockets to address connection timeouts
+- Replace `Nullable(const T&)` with `Nullable(T)` to avoid extra copy when initialized with an rvalue.
```

Comment:
```suggestion
- Replace `Nullable(const T&)` with `Nullable(T)` to avoid extra copy when initialized with an r-value.
```

Reply:
C++ refers to them by rvalue, lvalue.  Its uncommon to use hyphen syntax.